smithy-model/src/main/java/software/amazon/smithy/model/shapes/SmithyIdlModelSerializer.java

```diff
@@ -389,7 +389,9 @@ public SmithyIdlModelSerializer build() {
         @Override
         protected Void getDefault(Shape shape) {
             serializeTraits(shape);
-            codeWriter.write("$L $L", shape.getType(), shape.getId().getName()).write("");
+            codeWriter.writeInline("$L $L ", shape.getType(), shape.getId().getName());
```

Comment:
I'm guessing CodeWriter is handling trimming spaces at the end of lines, right?

Reply:
yes